companies/services/report_section_extractor.py

```
@@ -0,0 +1,246 @@
+            if "사업보고서" in report_name or "반기보고서" in report_name:
+                return self._extract_business_sections(soup, report_name)
+            else:
+                # 기타 보고서는 전체 내용 사용
+                return self._extract_all_content(soup)
+
+        except Exception as e:
+            logger.error(f"섹션 추출 실패: {e}", exc_info=True)
+            return {"main_content": "", "tables": ""}
+
+    def _extract_business_sections(
+        self, soup: BeautifulSoup, report_name: str
```

Ruff (0.14.13)

Unused method argument: `report_name`

(ARG002)